docs/api-testing/contract-testing.md

```diff
@@ -0,0 +1,55 @@
+---
+id: contract-testing
+title: API Contract Testing
+sidebar_label: API Contract Testing
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+An API conversation consists of:
+* API consumer (client side) performing a request
+* API producer (server side) responding to that request<br/><img src={useBaseUrl('img/api-fortress/2022/03/api-conversation.png')} alt="API Conversation and Contract" width="300"/>
+
+The conversation needs to follow specific rules that the API producer and API consumer must agree upon. The formal description of these rules is the contract, which is generally presented as a specification file such as [OpenAPI](https://swagger.io/docs/specification/about/).
+
+If this contract is broken by either party, it can lead to bugs and malfunctions. _API Contract Testing_ is the act of validating that the API producer and the API consumer are respecting the contract. That’s where Sauce Labs API Testing comes in.
+
+## What You'll Need
+* A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up)).
+* An existing API Testing Project. For details on how to create one, see [API Testing Quickstart](/api-testing/quickstart/).
+
+
+## About Contract Testing  
+
+Contract testing is a fast, lightweight form of API testing that strictly checks the content and format of requests and responses. This method is ideal for:
+* Testing APIs during the early stages of design and development
+* Organizations creating APIs that are internal and/or have limited number of consumers
+
+This is typically done in a protected, static environment, where tests are run against mocked (not live) APIs, allowing contract tests to compare isolated API responses to the contract for immediate attention if something is wrong.<br/><img src={useBaseUrl('img/api-fortress/2022/03/api-consumer-contract.png')} alt="API Conversation and Contract" width="500"/>
+
+### Producer (Server) Side
+To test the producer (server) side:
+1. From an API Testing Project, go to the HTTP Client.
+2. Import an OpenAPI specification file (v3.0 or higher).
+3. From your list of **Snapshots**, choose the API call you'd like to test by clicking on it. The HTTP method, request URL, and anything else specified in the spec file will populate in the HTTP Client fields.
+4. Click **Generate Test**.
```

Comment:
@kimsauce before Generate Test you need to **Send** the request (snapshot) otherwise it will generate only the call